src/completed.js

```diff
@@ -0,0 +1,22 @@
+import { renderTodos } from './todo-utils.js'; // eslint-disable-line
+export const refresh = () => {
+  renderTodos(); // Now “renderTodos” can be called after it’s defined
+};
+export const getTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+export const updateStatus = (index, completed) => {
+  let todos = getTodos();
+  todos = todos.map((todo) => {
+    if (todo.index === index) {
+      todo.completed = completed;
+    }
+    return todo;
+  });
+  localStorage.setItem('todos', JSON.stringify(todos));
+  refresh();
+};
+export const clearCompleted = () => {
+  let todos = getTodos();
+  todos = todos.filter((todo) => !todo.completed);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  refresh();
+};
```

Comment:
You did well in implementing the  `interactive` feature, however, there seems to be an abnormal behavior in your app such that when I click on an item to be `cleared`, it selects two items which in turn would be deleted. Please this isn't the expected behavior and I'd kindly suggest you fix it 😄 

![image](https://github.com/Armynerh/todo-list-project/assets/20537712/fe56503b-add8-4cd6-a960-1b915428ec43)